### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "connect ETIMEDOUT 66.220.148.145:443"

### DISCUSSION THREAD

**Discussion by LuyuZhang00** [community/community#42586] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I cann‘t use it in vscode  for a week, How can I solve
`GitHub Copilot could not connect to server. Extension activation failed: "connect ETIMEDOUT 66.220.148.145:443"`

**Comment by jfullstackdev** [comment]
you can browse the category or search it, maybe there are similar topics same as yours

one thing is that, is it the latest version ? did you try downgrading ?

**Comment by iamcjwilliams** [comment]
@LuyuZhang00 Visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-visual-studio-code
